src/lib/components/molecules/fptp-waffle/index.jsx

```diff
@@ -0,0 +1,14 @@
+import { Waffle, HalfLineText } from '$particles'
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const FPTPWaffle = ({ total, units, rows, abbreviationAccessor, halfLineTextHeight, paddingTop, onMouseOver, squares, styles }) => {
```

Comment:
I would be in favour of implementing this as a wrapper component. It could be called `FirstPastThePostFurniture` or something like that. The component would be much simpler, because it doesn't need to pass all those props to the `Waffle` component.